datalake/src/main/java/com/sequenceiq/datalake/job/SdxClusterStatusCheckerJob.java

```diff
@@ -74,6 +74,10 @@ protected void executeTracedJob(JobExecutionContext context) {
 
     private void syncSdxStatus(JobExecutionContext context) {
         getCluster().ifPresent(sdx -> {
+            if (!getRemoteResourceCrn().equals(sdx.getStackCrn())) {
+                setRemoteResourceCrn(sdx.getStackCrn());
+            }
+
```

Comment:
Please add a comment with the explanation for the change.

Reply:
Done.